apps/whale-api/src/module.api/address.controller.ts

```diff
@@ -87,7 +87,7 @@ export class AddressController {
         const start = foundIndex + 1 // plus 1 to exclude the prev txid
         const size = start + query.size
         const sliced = list.slice(start, size)
-        if (sliced.length !== query.size) {
+        if (sliced.length === 0) {
```

Comment:
I'm a little confused, you mind explaining how it fixes it?

Reply:
yea sure.. should document it

Reply:
phew.. updated!!

Reply:
resolved and refer to https://github.com/JellyfishSDK/jellyfish/pull/1710#discussion_r950346813